uaclient/apt.py

```diff
@@ -14,6 +15,7 @@
     # typing isn't available on trusty, so ignore its absence
     pass
 
+APT_HELPER_TIMEOUT = 20.0    # 20 second timeout used for apt-helper call
```

Comment:
Absolutely no problems defining this as a float. But as a note, to make sure my previous comment wasn't misunderstood, an `int` is an acceptable value for a variable of type `float`:

```
$ echo 'x: float = 1' > test.py
$ mypy --strict test.py
$ echo $?
0
```

though the inverse is not true:

```
$ echo 'x: int = 1.0' > test.py
$ mypy --strict test.py
test.py:1: error: Incompatible types in assignment (expression has type "float", variable has type "int")
```

Reply:
Good point, and thanks for clarification, I wanted the float there as a hint of the type that is allowed in case we are tweaking that number at the declaration level and don't see that floats are allowed.